keyboards/ta65/keymaps/maartenwut/config.h

```diff
@@ -0,0 +1 @@
+#define RETRO_TAPPING
```

Comment:
This should have an include guard for this. 

```suggestion
#pragma once 

#define RETRO_TAPPING
```

Reply:
thanks Drashna. I wasn't sure if it needed it because of how it's included